coding-standards/html-css/README.md

```diff
@@ -1,40 +1,93 @@
+    display: block; z-index: 100; left: 0;
+}
+```
+
+By grouping and ordering properties in a consistent manner, we can see at a glance what the element will look like and how it's positioned. This reduces a lot of cognitive load.
+
+
+### Utility classes
+
+Avoid using utility classes, especially for overall styling. It will inevitably lead to unforseen bugs and unnecessary complexity.
+
+- It creates unnecessary cognitive load, since you might have to combine the properties of 5 classes in your head to get the complete 'selector'.
+- Utility classes are usally combined, resulting in specificity issues. The order of the appended class does not matter, only the cascade in our stylesheet dictates which property 'wins'.
+- Some developers might be more familiar with them than others. This easilly leads to bugs, since changing the styling for element A, also changes the styling for element B, F and Y.
+- You might want to have a small variation since element C needs to look slightly different than element B, leading to possibly many variations of a simple utility class.
+- It creates the false illusion that a (back-end) developer can create the styling for a newly created feature by glueing utility classes together. This sometimes works. It sometimes simply does not.
+
+Can you tell what the following element will look like?
+
+```html
+<div class="text block widget widget--small margin-top left"></div>
+```
+
+You might have a general idea. But you still would need to look up all those classes, and combine them in your head to get the full picture. Furthermore, changing `.text` for this specific element, might break another element using that same `.text`.
+
+```html
+<div class="specific-module"></div>
+```
+
+Using a specific module class for our element, means that all our styling is scoped to that module. This means that we can look up `.specific-module` and be sure that everything is in there. Changing that module will only change (and possibly break) that module.
+
+Note: using scoped modules means that we sometimes might have duplicate groups of properties, which are shared between multiple modules. This however, does not wheigh up to the fact that it's a lot safer to work with scoped modules. Also variation and exceptions are fairly common, which means that only a subset is usally shared between modules. Furthermore, you can always write a simple mixin and import that in your module. Beware that this can lead to generic 'utility'-mixins as well, and only shifts the cause of the problem elsewhere.
 
 ### IDs in CSS
 
 Never use IDs in CSS. Ever. They have no advantage over classes (anything you can do with an ID, you can do with a class), they cannot be reused, and their specificity is way, way too high. Even an infinite number of chained classes will not trump the specificity of one ID.
+Note: this does not mean that we don't use IDs in our markup, since they are used for accessibility purposes (eg. `aria-controlledby="the-other-id"`).
```

Comment:
Perhaps worth mentioning: also to create linkable anchors.

Reply:
👍